### PR TITLE
Buttons

### PR DESCRIPTION
**Pull request by shivampandey0**
Buttons:

https://user-images.githubusercontent.com/51923995/151958097-3ab8135b-0369-4cd4-ba6c-5a0b1e1eb4a1.mov

- Primary
- Secondary
- Outlined
- Icon Button
- Floating Action Button

### CODE REVIEW

File: css/index.html [hunk 1]
```diff
@@ -98,7 +111,6 @@ <h2 class="title">Avatars</h2>
             </div>
         </section>
     </main>
-
 </body>
 
-</html>
+</html>
```
Comment (Hamdrive) [inline css/index.html]: Add newline to overcome this error
Reference: [https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log](url)

**Review by Hamdrive — COMMENTED** [review]
Looks good to me!
Clean code

File: css/elements/buttons.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+.btn {
+  background-color: var(--text-white);
+  border: 1px solid transparent;
+  color: var(--text-black);
+  cursor: pointer;
+  padding: 0.5rem 1rem;
+
+  font-size: 1rem;
+  text-align: center;
```
Comment (MaruthiPothuganti) [inline css/elements/buttons.css]: Text will be centered implicitly

**Review by raghav2110 — COMMENTED** [review]
Overall code looks good to merge...Just add that extra line as Hamza said.

**Review by MaruthiPothuganti — COMMENTED** [review]
One change, Rest looks good

**Pull request merged**